### ISSUE TITLE
Collection variable reference is not showing up

### ISSUE THREAD

**Issue by ashishtilara** [a85/POSTMan-Chrome-Extension#13159]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues and I know that duplicates will be closed

### Describe the Issue

Postman:
Version 11.14.1
UI Version: 11.14.1-ui-240925-1856
Desktop Platform Version: 11.14.0 (11.14.0)

Collection variables are not resolving

### Steps To Reproduce

I have collection variables
<img width="982" alt="image" src="https://github.com/user-attachments/assets/623711b8-5163-418b-b2be-53164fe4cd32">

when I try to use and check the value in request body
<img width="507" alt="image" src="https://github.com/user-attachments/assets/a8753f4d-a249-4b92-a73a-458f920f7130">

or URL
<img width="696" alt="image" src="https://github.com/user-attachments/assets/89cb1725-a27e-4e9a-9253-63dcc28d9459">

it's not resolving to correct value.

### Operating System

macOS

### Postman Version

11.14.1

### Postman Platform

Postman App

### User Account Type

Signed In User

**Comment by skatikia** [comment]
@ashishtilara This should not be happening. 🙁 

Can you make sure that the collection variable is saved? Cmd+S or click the save button in the top right corner of the collection interface. It's silly, but I'm very prone to it. So, just making sure.

**Comment by ashishtilara** [comment]
@skatikia yes they were saved for sure. It was working in previous version as well.

**Comment by ronork** [comment]
Hey @ashishtilara, Thanks for reaching out! We couldn’t reproduce the issue on our end. Could you check if the issue occurs with other collections or is specific to that one? If it's only happening with that collection, please share the app logs. You can find instructions here: [How to get logs from the Postman Desktop app](https://support.postman.com/hc/en-us/articles/360025298633-How-to-get-logs-from-the-Postman-Desktop-app#h_01G396RG1PATWM520KV442RCSJ).

If it’s happening for all collections, please provide the steps you followed or a video if that’s easier, so we can investigate further.

**Comment by ashishtilara** [comment]
@ronork its all collections under `APIs` section.

**Comment by ronork** [comment]
Hey @ashishtilara, We’re still unable to reproduce the issue. Could you share a quick video of your workflow so we can check what we might be missing? Alternatively, you can [schedule a call ](https://calendly.com/d/cqb3-f3r-qvt/variable-issue-debug) to walk us through it.

**Comment by afripoint-eedris** [comment]
Facing this issue on the Postman VS Code extension. I create a new collection, defined variables in the collection, but can't get a reference to them.
